crates/node/core/Cargo.toml

```
@@ -77,4 +77,5 @@ jemalloc = ["reth-cli-util/jemalloc"]
 asm-keccak = ["reth-primitives/asm-keccak", "alloy-primitives/asm-keccak"]
 
 [build-dependencies]
-vergen = { workspace = true, features = ["build", "cargo", "git", "gitcl"] }
+vergen = { workspace = true, features = ["build", "cargo"] }
+vergen-git2 = {workspace = true}
```

```suggestion
vergen-git2.workspace = true
```
use this syntax if setting an individual variable